yarn.lock

```diff
@@ -2,67 +2,13 @@
 # yarn lockfile v1
```

Comment:
Can you revert yarn.lock. There isn no package addition/deletion/updation, so it must not be changed.